apertium-crh.crh.lexc

```diff
@@ -8,65 +9,85 @@
+
 Multichar_Symbols 
 
-%<n%>             ! Noun                        !
-%<np%>            ! Proper noun                 !
-%<adj%>           ! Adjective                   !
-%<prn%>           ! Pronoun                     !
-%<post%>          ! Postposition                !
-%<adv%>           ! Adverb                      !
-%<v%>             ! Verb                        !
-%<cnjcoo%>        ! Coordinating conjunction    !
-%<cnjsub%>        ! "subordinating conjunction" (ki)
-%<cnjadv%>        ! 
+%<n%>             ! Noun                        
+%<np%>            ! Proper noun                 
+%<adj%>           ! Adjective                   
+%<prn%>           ! Pronoun                     
+%<post%>          ! Postposition                
+%<adv%>           ! Adverb                      
+%<v%>             ! Verb                        
+%<cnjcoo%>        ! Coordinating conjunction    
+%<cnjsub%>        ! Sub-ordinating conjunction
+%<cnjadv%>        ! Adverbial conjunction 
 %<qst%>           ! Question word
-%<sym%>
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
 %<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<det%>           ! Determiner
+%<num%>           ! Numeral
 %<vaux%>          ! Auxiliary verb
 %<abbr%>          ! Abbreviation
-%<emph%> 
-%<ij%>
-%<nom%>           ! Nominative                  !
-%<gen%>           !
-%<dat%>           !
-%<acc%>           !
-%<loc%>           ! Locative                    ! 
-%<abl%>           ! Ablative                    !
-%<ins%>           !
-%<sim%>
-
-%<iv%>            !
-%<tv%>            !
+%<emph%>          ! Emphatic [imperative]
+%<ij%>		  ! Interjection
+%<part%>          ! Particle 
+
+! Proper noun types
+%<top%>           ! Toponym (place name)
+%<ant%>           ! Anthroponym (person name)
+%<cog%>           ! Cognomen (family name)
+%<org%>           ! Organisation
+%<pat%>           ! Patronymic
+%<al%>            ! Others
+
+! gender !
+%<m%>             ! Masculine    
+%<f%>             ! Feminine     
+%<mf%>            ! Either
+
+! Cases !
+
+%<nom%>           ! Nominative
+%<gen%>           ! Genitive
+%<dat%>           ! Dative
+%<acc%>           ! Accusative
+%<abl%>           ! Ablative
+%<loc%>           ! Locative
+%<ins%>           ! Instrumental=comitative 
+%<evid%>
 
-%<nec%>
-%<imp%>
-%<aor%>
+%<nec%>           ! Necessitive
+%<aor%>           ! Aorist
+%<imp%>           ! Imperative
+%<emph%>          ! Emphatic [imperative]
+%<cond%>          ! Conditional  ! Шарт  ! Условное  !! -с{A}
+%<caus%>          ! Causative
+%<pass%>          ! Passive
+%<coop%>          ! Cooperative
+%<rec%>           ! Reciprocal (?)
+%<iver%>	         ! suffix conveying ease/speed of action
+%<eyaz%>	         ! suffix conveying the action *almost* happened
 %<gno%>
 %<fut%>
-%<fut2%>          ! -EcAG
-%<past%>
-%<ifi%>
-
-%<neg%>
-
-%<pass%>
-%<coop%>
-%<caus%>
-%<abil%>
-%<evid%>
+%<fut2%>
+%<sim%>
```

Comment:
Is this a verb tense?

Reply:
I dont know abt these tags so i left them